packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx

```diff
@@ -0,0 +1,250 @@
+  localeStrings: Translations;
+}
+
+const chipColorMap = {
+  [RenewalStatus.AUTOMATIC]: OdsThemeColorIntent.success,
+  [RenewalStatus.AUTO]: OdsThemeColorIntent.success,
+  [RenewalStatus.MANUAL]: OdsThemeColorIntent.warning,
+  [RenewalStatus.BILLING_SUSPENDED]: OdsThemeColorIntent.info,
+  [RenewalStatus.MANUAL_FORCED]: OdsThemeColorIntent.info,
+  [RenewalStatus.DELETE_AT_EXPIRATION]: OdsThemeColorIntent.error,
+  [RenewalStatus.EXPIRED]: OdsThemeColorIntent.error,
+  [RenewalStatus.UNKNOWN]: OdsThemeColorIntent.error,
+};
+
+@Component({
+  tag: 'msc-billing-renewal',
+  styleUrl: 'msc-billing-renewal.scss',
+  shadow: true,
+})
+export class MscBillingRenewal implements IMSCBillingRenewal {
+  @Prop() public renewLinkDataTracking?: string;
+
+  @Prop() public cancelResiliationDataTracking?: string;
+
+  @Prop() public manageRenewDataTracking?: string;
+
+  @Prop() public resiliateDataTracking?: string;
+
+  @Prop() public anticipateRenewDataTracking?: string;
+
+  @Prop() serviceName: string;
+
+  @Prop() serviceType: string;
+
+  @Prop() servicePath: string;
+
+  @Prop() nextBillingDate?: string;
+
+  @Prop() serviceInfos?: ServiceInfos;
+
+  @Prop() serviceDetails?: ServiceDetails;
+
+  @Prop() localeStrings: Translations;
+
+  private getRenewStatus() {
+    if (this.serviceInfos?.status === 'ok') {
+      if (this.serviceInfos.renew.deleteAtExpiration) {
+        // Red chip 'Cancellation requested', link in menu 'Stop cancellation of service'
+        return RenewalStatus.DELETE_AT_EXPIRATION;
+      }
+      // service still active
+      if (this.serviceInfos.renew.automatic) {
+        // Green chip 'Automatic renewal', link in menu 'Manage my commitment' and 'Cancel subscription'
+        return RenewalStatus.AUTOMATIC;
+      }
+      if (this.serviceInfos.renew.manualPayment) {
+        // Yellow chip 'Manual renewal'
+        return RenewalStatus.MANUAL;
+      }
+      if (this.serviceInfos.renew.forced) {
+        return RenewalStatus.MANUAL_FORCED;
+      }
+    }
+    if (this.serviceInfos?.status === 'expired') {
+      // Red chip 'expired', link in menu 'Renew service'
+      return RenewalStatus.EXPIRED;
+    }
+    if (this.serviceDetails?.ressource?.state === 'suspended') {
+      return RenewalStatus.BILLING_SUSPENDED;
+    }
+    return RenewalStatus.UNKNOWN;
+  }
+
+  private getMenuLink() {
+    switch (this.getRenewStatus()) {
+      case RenewalStatus.EXPIRED:
+        return (
+          <osds-link
+            data-tracking={this.renewLinkDataTracking}
+            color={OdsThemeColorIntent.primary}
+            href={getRenewUrl(this.serviceName)}
+            target={OdsHTMLAnchorElementTarget._blank}
+          >
+            {this.localeStrings.billing_services_actions_menu_renew}
+            <osds-icon
+              class="new-link-icon"
+              size={OdsIconSize.xxs}
+              name={OdsIconName.EXTERNAL_LINK}
+              color={OdsThemeColorIntent.primary}
+            />
```

Comment:
**nitpick:** Add an input to ODS icon equivalent to `aria-hidden="true"`

Reply:
It is automatic on ODS icons